components/collective-page/sections/Contribute.js

```diff
@@ -163,6 +164,7 @@ class SectionContribute extends React.PureComponent {
     const hasNoContributor = !this.hasContributors(contributors);
     const canContribute = collective.isActive && !isPastEvent(collective);
     const hasCustomContribution = !collective.settings?.disableCustomContributions;
+    const hasCryptoContribution = collective.settings?.disableCryptoContributions === false;
```

Comment:
Don't we want to enable crypto only for specific hosts (OSC)?

Reply:
Ops, that's correct. I've added a condition to make it only available for OSC for now. 🙂